### ISSUE TITLE
Use `pino` instead of `console.log`

### ISSUE THREAD

**Issue by NimJay** [GoogleCloudPlatform/microservices-demo#674]
### Describe request or inquiry 

* Both JavaScript/Node.js microservices (`paymentservice` and `currencyservice`) use `console.log`.
* See files:
  * [`/src/paymentservice/index.js`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/80acbf22ce074dce22e03dd38efbf156f88eae99/src/paymentservice/index.js#L21)
  * [`/src/currencyservice/server.js`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/80acbf22ce074dce22e03dd38efbf156f88eae99/src/currencyservice/server.js#L18)
* Let's use `pino` instead.
* We're already using `pino` in other JavaScript files.
* This idea originated from [665](https://github.com/GoogleCloudPlatform/microservices-demo/pull/665/files#r774204647). 

### What purpose/environment will this feature serve? 

* See [pull-request 66](https://github.com/GoogleCloudPlatform/microservices-demo/pull/66) — where the use of `pino` was introduced — for reasoning. 
* Basically, we want our logs to be formatted as JSON — for [Stackdriver](https://cloud.google.com/products/operations). See #47.

**Comment by NimJay** [comment]
## Update (2022 July)
Just adding an update to cool this issue down (i.e., remove it from our team's out-of-SLO list).
No progress updates and this issue it still very low priority.
I wonder if this ticket will be addressed (indirectly) by https://github.com/GoogleCloudPlatform/microservices-demo/issues/422.

**Comment by aarushisoni** [comment]
Hi I want to contribute to this issue please assign me this issue

**Comment by NimJay** [comment]
Hi @aarushisoni,
Thanks again for your interest.
This is definitely a good first issue to work on. :)
I've assigned it to you.

**Comment by TonyBhargav** [comment]
Hi @NimJay since no progress in this issue can I get assigned to this issue. I'll be happy to work on this.
@aarushisoni are you still working on this?

**Comment by andresmc98** [comment]
Hi @NimJay 
is someone working on this issue? i would like give it a shot c:

**Comment by NimJay** [comment]
Hi @andresmc98, :)

I don't think anyone was working on this. Correct my if I'm wrong, @TonyBhargav. :)
I just assigned it to you.

Just a heads-up: please see before starting any work:
* [`CONTRIBUTING.md`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/CONTRIBUTING.md)
* [`development-guide.md`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/docs/development-guide.md)

Thank you all, for your interest in contributing.

**Comment by andresmc98** [comment]
Thank you,
so i'll checks the docs first

**Comment by DracFiendMG** [comment]
Hi @NimJay, can you assign this one to me? I've already made relevant changes to both **index.js** and **server.js** files. I want to confirm the changes with you.

**Comment by DracFiendMG** [comment]
@NimJay, I just noticed that **server.js** already has pino implemented, can you confirm?

**Comment by DracFiendMG** [comment]
@NimJay, I've raised a PR for this issue: https://github.com/GoogleCloudPlatform/microservices-demo/pull/2262
Can you look into it and let me know if I'm missing something?

**Comment by DracFiendMG** [comment]
@NimJay, I've already committed my changes can you look into it
